code/game/objects/items/weapons/weaponry.dm

```diff
@@ -28,6 +28,7 @@
 	force = 2
 	throwforce = 1
 	w_class = 3
+	knockout = 100

```

Comment:
if this isn't already defined you need to define it on the /obj/item definition.